projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -500,58 +539,64 @@ static function skip_stylesheet() {
 
 		if ( null !== $skip_stylesheet ) {
 			return $skip_stylesheet;
-		} elseif ( Jetpack_Custom_CSS::is_customizer_preview() ) {
+		} elseif ( self::is_customizer_preview() ) {
 			return false;
 		} else {
-			if ( Jetpack_Custom_CSS::is_preview() ) {
-				$safecss_post = Jetpack_Custom_CSS::get_current_revision();
+			if ( self::is_preview() ) {
+				$safecss_post = self::get_current_revision();
 
-				if ( $safecss_post )
-					return (bool) ( get_post_meta( $safecss_post['ID'], 'custom_css_add', true ) == 'no' );
-				else
-					return (bool) ( get_option( 'safecss_preview_add' ) == 'no' );
-			}
-			else {
-				$custom_css_post_id = Jetpack_Custom_CSS::post_id();
+				if ( $safecss_post ) {
+					return (bool) ( get_post_meta( $safecss_post['ID'], 'custom_css_add', true ) === 'no' );
+				} else {
+					return (bool) ( get_option( 'safecss_preview_add' ) === 'no' );
+				}
+			} else {
+				$custom_css_post_id = self::post_id();
 
 				if ( $custom_css_post_id ) {
 					$custom_css_add = get_post_meta( $custom_css_post_id, 'custom_css_add', true );
 
 					// It is possible for the CSS to be stored in a post but for the safecss_add option
 					// to have not been upgraded yet if the user hasn't opened their Custom CSS editor
 					// since October 2012.
-					if ( ! empty( $custom_css_add ) )
+					if ( ! empty( $custom_css_add ) ) {
 						return (bool) ( $custom_css_add === 'no' );
+					}
 				}
 
-				return (bool) ( Jetpack_Options::get_option_and_ensure_autoload( 'safecss_add', '' ) == 'no' );
+				return (bool) ( Jetpack_Options::get_option_and_ensure_autoload( 'safecss_add', '' ) === 'no' );
 			}
 		}
 	}
 
-	static function is_preview() {
-		return isset( $_GET['csspreview'] ) && $_GET['csspreview'] === 'true';
+	/**
+	 * Checks if we're in a preview mode.
+	 */
```

Comment:
```suggestion
	 *
	 * @return bool
	 */
```
Same or similar for several below too. If it does `return` with a value, we should document what types it returns.